src/facade/dragonfly_listener.cc

```diff
@@ -156,25 +156,30 @@ bool ConfigureKeepAlive(int fd) {
   return true;
 }
 
-thread_local size_t ssl_allocated_bytes = 0;
+struct ListenerStats {
+  size_t ssl_allocated_bytes = 0;
```

Comment:
```suggestion
  size_t tls_allocated_bytes = 0;
```
We use the term TLS (mostly?) instead of SSL

Reply:
ironically you introduced ` ssl_allocated_bytes` :)

Reply:
lol you're right
Now that you mention it, I remember thinking that it should be exposed "externally" (to the file) as TLS but internally since it's using SSL* functions it should be named as such. But since it confused me.. yeah, it's good that you changed it. Sorry for having named it as such to begin with!